Volume/View Models/ArticleInfo.swift

```diff
@@ -13,36 +13,37 @@ struct ArticleInfo: View {
 
     let article: Article
     let showsPublicationName: Bool
-    let largeFont: Bool
+    let isDebrief: Bool
 
     var body: some View {
         HStack {
             VStack(alignment: .leading, spacing: 0) {
                 if showsPublicationName {
                     Text(article.publication.name)
-                        .font(largeFont ? .newYorkMedium(size: 16) : .newYorkMedium(size: 12))
-                        .padding(.bottom, largeFont ? 3.5 : 1.5)
+                        .font(isDebrief ? .newYorkMedium(size: 18) : .newYorkMedium(size: 12))
```

Comment:
similar to noah's suggestion: 
`.newYorkMedium(size: isDebrief ? 18 : 12)`

Reply:
Yup, thanks Amy!